react/components/ProductAvailability.tsx

```diff
@@ -3,19 +3,24 @@ import type { ProductTypes } from 'vtex.product-context'
 
 import LowStock from './LowStock'
 import HighStock from './HighStock'
+import ShowAvailable from './ShowAvailable'
 import Container from './Container'
 
 interface Props {
   threshold: number
   lowStockMessage?: string
   highStockMessage?: string
+  showAvailability?: boolean
```

Comment:
We should avoid boolean props as much as possible since they don't scale very well. I left my suggestion below but feel free to use better names if you can think of any. Also, when you change this, don't forget to update the docs as well :)

```suggestion
  showAvailability?: 'threshold' | 'quantity'
```